packages/react-components/react-input/src/components/Input/useInputStyles.ts

```diff
@@ -132,7 +126,9 @@ const useRootStyles = makeStyles({
   },
   underline: {
     backgroundColor: tokens.colorTransparentBackground,
-    ...shorthands.borderRadius(0), // corners look strange if rounded
+    // corners look strange if rounded
+    borderBottomLeftRadius: 0,
+    borderBottomRightRadius: 0,
```

Comment:
You should probably use

```suggestion
    borderBottomLeftRadius: tokens.borderRadiusNone,
    borderBottomRightRadius: tokens.borderRadiusNone,
```

Reply:
It was using 0 before. I'm actually just going to revert this change and the borderRadius change below, since it's unrelated to the main change in this PR.